src/wrappers/themis/jsthemis/secure_cell_context_imprint.cpp

```diff
@@ -42,6 +43,21 @@ namespace jsthemis {
 
   void SecureCellContextImprint::New(const Nan::FunctionCallbackInfo<v8::Value>& args) {
     if (args.IsConstructCall()) {
+      if(args.Length()<1){
+        ThrowError("Secure Cell (Context Imprint) constructor", THEMIS_INVALID_PARAMETER, "missing master key");
+        args.GetReturnValue().SetUndefined();
```

Comment:
Uh... Cargo cult? I'm not really sure myself that we have to do this. But this is how it was before these changes:

https://github.com/cossacklabs/themis/blob/463f105bec866bd0cee6ceed4f3cdcb2320bcc27/src/wrappers/themis/jsthemis/secure_message.cpp#L65-L69

So I just kept the pattern: throw, set undefined, return.

I imagine that the default return value is already `undefined` and that we don't really have to set it because we're throwing an exception instead of returning normally. But I don't know. It's kinda bothersome to write, but I'm no V8 export to tell for sure that i don't need to.

Reply:
okay-okay :)
I thought maybe you know about some existing conventions in node.js world